DataConnectors/SentinelOne/SentinelOneSentinelConnector/__init__.py

```diff
@@ -39,14 +43,31 @@ def __init__(self):
         self.from_date, self.to_date = self.generate_date()
         self.results_array = []
         self.threat_id_arr = []
+        self.computerName_arr = []
 
     def get_threat_id(self, json_obj_arr):
+        """

```

Comment:
Use docstring format for the function comments

Reply:
I don't understand, I thought it is docstring. What part do I have to change?

Reply:
ideally it should have description, param names and return type, etc.

Reply:
Example:
ef square_root(n):
    """Calculate the square root of a number.

    Args:
        n: the number to get the square root of.
    Returns:
        the square root of n.
    Raises:
        TypeError: if n is not a number.
        ValueError: if n is negative.

    """
    pass

Reply:
oh okay understood. Will add the changes to all functions.